docs/advanced/customization-guide.md

```diff
@@ -74,6 +74,57 @@ Now, on X86 platforms the feature label appears after doing `modprobe dummy` on
 a system and correspondingly the label is removed after `rmmod dummy`. Note a
 re-labeling delay up to the sleep-interval of nfd-worker (1 minute by default).
 
+### Taints
```

Comment:
We could declare this as *EXPERIMENTAL*, dunno 🤔 This would keep the door better open for future adjustments if needed 😅 Any thoughts?

Reply:
+1